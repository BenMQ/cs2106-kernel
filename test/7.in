cr a 2
req R4 4
cr b 2
to
req R3 3
cr c 2
cr d 2
to
to
req R3 1
req R3 1
to
de a
cr a 2
req R4 4
cr b 2
to
req R3 3
cr c 2
cr d 2
to
to
req R3 1
req R3 1
to
