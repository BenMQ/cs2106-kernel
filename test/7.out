init a a a b b b b a c d b a init a a a b b b b a c d b a 
